ios/Podfile.lock

```diff
@@ -331,6 +331,6 @@ SPEC CHECKSUMS:
   url_launcher_ios: 68d46cc9766d0c41dbdc884310529557e3cd7a86
   workmanager: 0afdcf5628bbde6924c21af7836fed07b42e30e6
 
-PODFILE CHECKSUM: 91d9116a6c482e0d0b230a68097981fae7e4d07a
+PODFILE CHECKSUM: ded4724b53568389542fa0f7d7a64c05ffc03971
```

Comment:
Please run again `pod install ` to update Podfile.lock for `app_settings`

Reply:
Done